types.d.ts

```diff
@@ -0,0 +1,101 @@
+declare module "logdna" {
+  type Level = "Debug" | "Trace" | "Info" | "Warn" | "Error" | "Fatal";
```

Comment:
This should maybe be a `string`? If any custom level is supported.

Reply:
Yes, please change it to string. We should be able to add a custom level in the options object.

Reply:
Thanks @vilyapilya, I have removed this type and set the level type to string. See: https://github.com/logdna/nodejs/pull/71/commits/ce78ca83e9cd3701d7794eb12e8324af37eab2c0